lib/rmg/CRmgTemplateZone.cpp

```diff
@@ -1463,6 +1475,38 @@ bool CRmgTemplateZone::createRequiredObjects()
 		}
 	}
 
+	//create nearby objects (e.g. extra resources close to mines)
+	for(const auto &object : nearbyObjects)
```

Comment:
space should be around & and * everywhere except when & is unary operator to take address